### PR TITLE
Feature/unit tests

### PR DESCRIPTION
**Pull request by warbhe-amit**
added test cases

### CODE REVIEW

File: server/test/suite/config.test.mjs [hunk 1]
```diff
@@ -0,0 +1,44 @@
+"use strict";
+import { config } from "../../src/config.mjs";
+
+test("should first", () => {
```
Comment (michaelcfine) [inline server/test/suite/config.test.mjs:4]: I'll defer to Anushri, but these kinds of tests seem like they're fragile and not worth writing. I think it's okay to leave it, I just don't see the benefit, really.

File: server/test/suite/events.test.mjs [hunk 2]
```diff
@@ -0,0 +1,130 @@
+"use strict";
+
+import { jest } from "@jest/globals";
+import * as events from "../../src/events.mjs";
+import { logger } from "../../src/logger.mjs";
+
+test("events.registerEventListener is working properly", () => {
+  const spy = jest.spyOn(logger, "debug");
+  const dumyObject = {
```
Comment (michaelcfine) [inline server/test/suite/events.test.mjs:9]: "Dummy" has two Ms; if you're in the code making other changes, you can/should change, but it's obviously not high priority.

File: server/test/suite/fireboltOpenRpc.test.mjs [hunk 3]
```diff
@@ -0,0 +1,42 @@
+"use strict";
+import { jest } from "@jest/globals";
+import * as fireboltOpenRpc from "../../src/fireboltOpenRpc.mjs";
+
+test("fireboltOpenRpc.getRawMeta working properly", () => {
+  const expectedResult = {};
+  const result = fireboltOpenRpc.getRawMeta();
+  expect(result).toEqual(expectedResult);
```
Comment (michaelcfine) [inline server/test/suite/fireboltOpenRpc.test.mjs:8]: How does this work? Doesn't seem like it should...

**Review by michaelcfine — APPROVED** [review]
Looks good, although I didn't examine super closely. Let's get code coverage working so we can see where we're at.

File: server/test/suite/events.test.mjs (hunk 2 above)
Comment (warbhe-amit) [reply server/test/suite/events.test.mjs]: done

File: server/test/suite/fireboltOpenRpc.test.mjs (hunk 3 above)
Comment (warbhe-amit) [reply server/test/suite/fireboltOpenRpc.test.mjs:8]: In this fireboltOpenRpc.mjs file, getRawMeta function is returning an object which can't be accessible from test files as its blank object initially, that's why its returning an empty object and we are just checking with deep equality.
Please confirm.

File: server/test/suite/fireboltOpenRpc.test.mjs [hunk 4]
```diff
@@ -0,0 +1,735 @@
...
+    "invalidMethod",
+  ];
+  const expectedOutput = [true, true, false];
+  expectedInput.forEach((methodName, index) => {
+    const result = fireboltOpenRpc.isMethodKnown(methodName);
+    expect(result).toBe(expectedOutput[index]);
+  });
+});
+
+// test('fireboltOpenRpc.getSchema', () => {
```
Comment (warbhe-amit) [inline server/test/suite/fireboltOpenRpc.test.mjs:292]: getSchema_error

@michaelcfine, getting this error while running the test case.

**Pull request merged**